games/TicTacToe/README.md

```diff
@@ -30,28 +30,41 @@ The project consists of a smart contract.
 
 ```sh
 TicTacToe
-├── project
-│   ├── contracts
-│   │   └── tictactoe-contract
-│   │       ├── src/main.sw
-│   │       └── tests/harness.rs
-│   ├── README.md
-│   └── SPECIFICATION.md
-├── ui
-│   ├── README.md
-│   └── SPECIFICATION.md
+├── packages
+├────── project
+│   ├────── contracts
+│   │   └────── tictactoe-contract
+│   │       ├────── src/main.sw
+│   │       └────── tests/harness.rs
+│   ├────── README.md
+│   └────── SPECIFICATION.md
+├────── app
+│   ├────── package.json
+│   ├────── index.html
+│   └────── src/main.tsx
 └── README.md
 ```
 
 ## Running the project
 
 ### User interface
 
-TODO: The user interface does not currently exist therefore its [README.md](ui/README.md) and [SPECIFICATION.md](ui/SPECIFICATION.md) are empty.
+In order to run the subsequent commands change into the following directory `/path/to/TicTacToe/packages/app/<here>`.
+
+#### Run the frontend application
+
+```bash
+pnpm install
```

Comment:
Could we reduce the necessary commands to just install and spin up, or maybe even just a single command, via `package.json` scripts?
I've used something like the below before:
```json
"scripts": {
		"build:resources": "tsx generatePredicateResources.ts",
		"build": "run-s build:resources && tsc && vite build",
```